lib/tasks/serve.js

```diff
@@ -145,8 +145,10 @@ exports.start = function (mage, options, cb) {
 
 exports.shutdown = function (mage, options, cb) {
 	function closeServers(callback) {
-		mage.core.msgServer.close();
-		mage.core.httpServer.close(callback);
+		async.waterfall([
```

Comment:
On 2nd thought, @stelcheck just did a PR (#98) that splits `closeServers` up into two, so you don't have to worry about it if you pull develop.